config_loader/Berksfile

```diff
@@ -0,0 +1,18 @@
+source 'https://supermarket.chef.io'
+
+metadata
+
+# We have to reference this special citadel-build repository for two reasons:
+#
+# 1. We have a custom fork of https://github.com/poise/citadel at
+#    https://github.com/18f/citadel.git because of
+#    https://github.com/poise/citadel/pull/35 and
+#    https://github.com/poise/citadel/pull/36.
+# 2. There is an extra build step required, and the citadel repository itself is
+#    a valid cookbook.  See the citadel-build repository for more details.  This
+#    worked before without this extra repo because we were pulling the built
+#    citadel from https://supermarket.chef.io.
+#
+# Once the pull requests are merged and a new citadel version is released we can
+# get rid of this.
+cookbook 'citadel_fork', '~> 9.1.1', git: 'https://github.com/18f/citadel-build.git'
```

Comment:
We might want to add some documentation that callers need to put all the recursive dependencies in their toplevel Berksfile, since berkshelf won't process the Berksfile of dependency cookbooks.

We may also want to remove the dependency on citadel at some point, which never merged my pull requests and appears to be unmaintained.

Reply:
I'll copy/move over the comments from the previous location of these cookbooks.